packages/aws-cdk/lib/api/deploy-stack.ts

```diff
@@ -694,3 +694,9 @@ function suffixWithErrors(msg: string, errors?: string[]) {
     ? `${msg}: ${errors.join(', ')}`
     : msg;
 }
+
+export class StackNotFoundError extends Error {
+  constructor(message: string) {
+    super(message);
+  }
+}
```

Comment:
We don't need this error type here, please use `throw new Error('message');`

Reply:
Sure. Any reason why you feel this is unnecessary? Saw some custom errors in the code. Thought it would be make it clear which exception is being thrown.